README.rst

```diff
@@ -2,11 +2,11 @@
 Ichnaea
 =======
 
-``Ichnaea`` is an application to provide geolocation coordinates
+``Ichnaea`` is an application that provides geolocation coordinates
 from other sources of data (cell or WiFi networks, GeoIP, etc.).
 
-See `the full docs <https://mozilla.github.io/ichnaea/>`_
-for more information.
+For more information look at the full docs <https://mozilla.github.io/ichnaea/>`_
```

Comment:
I think you missed the beginning \` in front of `the full docs`. reStructuredText uses \`some text \<link\>\`_ to mark a link with some explicit link text.